internal/context/paths.go

```diff
@@ -54,7 +54,7 @@ func getProjectDir(configFileName string) (string, error) {
 		}
 	}
 
-	return "", fmt.Errorf("packages can be installed locally only under project directory, use \"--global\"")
+	return workDir, nil
```

Comment:
I would add a command flag that prevents from creating `klio.yaml` implicitly. Basing on experience, I made more than couple `klio.yaml` without the intention to do so -_-`

Reply:
We already have a flag `--no-init` that according to description does exactly that: `prevent creating config file if not exist`
However it seems that an empty klio.yaml is added even when using this flag
Combining `--no-init` with `--no-save` results in `klio.yaml`  not being created

Instead of adding a flag I will fix `--no-init` to work as intended

Reply:
adding `--no-init` does not create a `klio.yaml` if it does not exist